app/views/lead_providers/partnership_csv_uploads/error_page.html.erb

```diff
@@ -0,0 +1,8 @@
+<% content_for :title, "Upload partnership csv" %>
+<!-- do we need a back button here? -->
+<% content_for :before_content, govuk_back_link( text: "Back", href: '#') %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Csv errors page</h1>
```

Comment:
What is this page meant to represent in the flow? That a file was uploaded correctly but there were errors in it?

Reply:
Yes, that is the next step